notebooks/features/causal_inference/Effects of Outreach Efforts.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%configure -f\n",
+    "{\n",
+    "  \"name\": \"synapseml\",\n",
+    "  \"conf\": {\n",
+    "      \"spark.jars.packages\": \"com.microsoft.azure:synapseml-core_2.12:0.10.1-62-dc969b75-SNAPSHOT\",\n",
+    "      \"spark.jars.repositories\": \"https://mmlspark.azureedge.net/maven\",\n",
+    "      \"spark.jars.excludes\": \"org.scala-lang:scala-reflect,org.apache.spark:spark-tags_2.12,org.scalactic:scalactic_2.12,org.scalatest:scalatest_2.12\",\n",
+    "      \"spark.yarn.user.classpath.first\": \"true\",\n",
+    "  }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "# Startup Investment Attribution - Understand Outreach Effort's Effect"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "![image-alt-text](https://camo.githubusercontent.com/4ac8c931fd4600d2b466975c87fb03b439ebc7f6debd58409aea0db10457436d/68747470733a2f2f7777772e6d6963726f736f66742e636f6d2f656e2d75732f72657365617263682f75706c6f6164732f70726f642f323032302f30352f4174747269627574696f6e2e706e67)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "A startup that sells software would like to know whether its outreach efforts were successful in attracting new customers or boosting consumption among existing customers. In other words, they would like to learn the treatment effect of each investment on customers' software usage.\n",
```

Comment:
This is super awesome, perhaps just add a quick cell above this that sets the stage a little more broadly - Just a sentence to say that this is a worked example of using feature X for the use case Y. We'll be pushing these to the azure docs eventually so consider that audience

Reply:
Thank you, Mark, it's a good idea, the quick cell can save reader's time and give them an overview what's this sample notebook is about.

I added this cell, please let me know if it makes sense. 
**This sample notebook aims to show the application of using SynapseML's DoubleMLEstimator for inferring causality using observational data.**